### PR TITLE
Dep Updates - Week of 10/31/2022 

### PR DESCRIPTION
**Pull request by jasonfu9966**
## Welcome to another round of USTC Dependency Updates!
---

### Items of Note
+ **Node v16 changes are not included in this update.**
+ [Link](https://app.circleci.com/pipelines/github/flexion/ef-cms/38991/workflows/a8f29245-040f-4d81-bd27-1f02faa1074a) to Circle run with a successful migration after updates.
    + it is from a different branch, but the changes are the same.
+ Terraform version did not have an update.
+ Docker cypress/base image did not have an update.

edit: downgraded `pdfjs-dist` to old version and added a few more minor updates

### Major Updates

+ babel-loader: ^9.1.0
+ jsdoc: ^4.0.0

### Minor/Patch Updates

+ aws-sdk: ^2.1248.0
+ esbuild: ^0.15.13
+ esbuild-css-modules-plugin: ^2.6.2
+ joi: ^17.7.0
+ luxon: ^3.1.0
+ react-number-format: ^5.1.1
+ react-select: ^5.6.0
+ @babel/core: ^7.20.2
+ @babel/preset-env: ^7.20.2
+ @types/jest: ^29.2.8
+ @types/lodash: ^4.14.188
+ @typescript-eslint/eslint-plugin: ^5.42.0
+ @typescript-eslint/parser: ^5.42.0
+ artillery: ^2.0.0-24
+ autoprefixer: ^10.4.13
+ dynamodb-admin: ^4.5.0
+ eslint-plugin-jsdoc: ^39.6.2
+ jsdom: ^20.0.2
+ sass: ^1.56.0
+ stylelint: ^14.14.1
+ stylelint-config-standard-scss: ^6.1.0

### Caveats Added/Removed

+ None

### CODE REVIEW

**Review by TomElliottFlexion — APPROVED** [review]

**Comment by rosiehsmith** [comment]
The `Setup Migration Infrastructure` [job](https://app.circleci.com/pipelines/github/flexion/ef-cms/38991/workflows/a8f29245-040f-4d81-bd27-1f02faa1074a/jobs/256466) looks awfully fast at 0 seconds for running a migration -  did you happen to force a migration via one of [these steps](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/additional-resources/blue-green-migration.md#manual-migration-steps)? I could be looking at the wrong thing, I'm used to some of these jobs having a longer amount of time when going through the migrate steps.

File: cypress-integration/integration/petitioner-updates-email.cy.js [hunk 1]
```diff
@@ -24,9 +24,10 @@ describe('Petitioner updates and verifies their email', () => {
   });
 
   it('petitioner should be able to change their email', () => {
+    const randomSuffix = parseInt(Math.random() * 100);
     goToMyAccount();
     clickChangeEmail();
-    changeEmailTo('petitioner9+test@example.com');
+    changeEmailTo(`petitioner9+test${randomSuffix}@example.com`);
```
Comment (rosiehsmith) [inline cypress-integration/integration/petitioner-updates-email.cy.js:30]: 🙌

**Comment by jasonfu9966** [comment]
> The `Setup Migration Infrastructure` [job](https://app.circleci.com/pipelines/github/flexion/ef-cms/38991/workflows/a8f29245-040f-4d81-bd27-1f02faa1074a/jobs/256466) looks awfully fast at 0 seconds for running a migration - did you happen to force a migration via one of [these steps](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/additional-resources/blue-green-migration.md#manual-migration-steps)? I could be looking at the wrong thing, I'm used to some of these jobs having a longer amount of time when going through the migrate steps.

hm no, I didn't force anything. but I did have this running in exp1 and exp2 simultaneously, so maybe that affected it? I had to do a lot of finagling in an attempt to get things to work properly.

**Comment by rosiehsmith** [comment]
> hm no, I didn't force anything. but I did have this running in exp1 and exp2 simultaneously, so maybe that affected it? I had to do a lot of finagling in an attempt to get things to work properly.

Cool, once you validate the updates with a forced migration, 👍 everything else looks great - nice job!

File: cypress-integration/integration/petitioner-updates-email.cy.js (hunk 1 above)
Comment (mmarcotte) [reply cypress-integration/integration/petitioner-updates-email.cy.js:30]: ➕ 💯 (or should i say ✖️ 💯?)

**Review by mmarcotte — APPROVED** [review]

**Review by codyseibert — APPROVED** [review]

**Comment by jasonfu9966** [comment]
[link to exp1 build](https://app.circleci.com/pipelines/github/flexion/ef-cms/39015/workflows/0401adc2-b35a-4015-ab95-84349e0b6812) with the same changes in a different branch

**Review by rgoshen — APPROVED** [review]

**Pull request merged**